app/src/main/res/layout/fragment_beneficiary_add_options.xml

```diff
@@ -1,119 +1,125 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
+<ScrollView
```

Comment:
We don't require scrollView in portrait mode and this approach is not a standard way of handling landscape mode. Create another layout resource for landscape mode, and instead of scrolling the view, show all those three icons in one line.

Reply:
understood, will create another layout for the landscape mode

Reply:
> We don't require scrollView in portrait mode and this approach is not a standard way of handling landscape mode. Create another layout resource for landscape mode, and instead of scrolling the view, show all those three icons in one line.

@jawidMuhammadi I have made a separate layout just as you had asked for and I have attached a screen recording to showcase the changes

https://user-images.githubusercontent.com/90026952/230792238-d445f914-33e2-4b9e-853a-12dfca777956.mp4